src/Cli/dotnet/commands/dotnet-add/dotnet-add-package/AddPackageParser.cs

```diff
@@ -3,59 +3,70 @@
-                              CommonLocalizableStrings.CommandInteractiveOptionDescription,
-                              Accept.NoArguments()
-                                    .ForwardAs("--interactive")),
-                Create.Option("--prerelease",
-                              CommonLocalizableStrings.CommandPrereleaseOptionDescription,
-                              Accept.NoArguments()
-                                    .ForwardAs("--prerelease")));
+            Description = LocalizableStrings.CmdPackageDescription
+        };
+
+        public static readonly Option VersionOption = new Option<string>(new string[] { "-v", "--version" },
+                              LocalizableStrings.CmdVersionDescription)
+        {
+            Argument = new Argument<string>(LocalizableStrings.CmdVersion)
+        }.ForwardAsSingle(o => $"--version {o}");
+
+        public static readonly Option FrameworkOption = new Option<string>(new string[] { "-f", "--framework" },
+                              LocalizableStrings.CmdFrameworkDescription)
+        {
+            Argument = new Argument<string>(LocalizableStrings.CmdFramework)
+        }.ForwardAsSingle(o => $"--framework {o}");
+
+        public static readonly Option NoRestoreOption = new Option<bool>(new string[] { "-n", "--no-restore" }, LocalizableStrings.CmdNoRestoreDescription);
+
+        public static readonly Option SourceOption = new Option<string>(new string[] { "-s", "--source" },
+                              LocalizableStrings.CmdSourceDescription)
+        {
+            Argument = new Argument<string>(LocalizableStrings.CmdSource)
+        }.ForwardAsSingle(o => $"--source {o}");
+
+
+        public static readonly Option PackageDirOption = new Option<string>("--package-directory", LocalizableStrings.CmdPackageDirectoryDescription)
+        {
+            Argument = new Argument<string>(LocalizableStrings.CmdPackageDirectory)
+        }.ForwardAsSingle(o => $"--package-directory {o}");
+
+        public static readonly Option InteractiveOption = new Option<bool>("--interactive", CommonLocalizableStrings.CommandInteractiveOptionDescription)
+            .ForwardAs("--interactive");
+
+        public static readonly Option PrereleaseOption = new Option<bool>("--prerelease", CommonLocalizableStrings.CommandPrereleaseOptionDescription)
+            .ForwardAs("--prerelease");
+
+        public static Command GetCommand()
+        {
+            var command = new Command("package", LocalizableStrings.AppFullName);
+
+            command.AddArgument(CmdPackageArgument);
+            command.AddOption(VersionOption);
+            command.AddOption(FrameworkOption);
+            command.AddOption(NoRestoreOption);
+            command.AddOption(SourceOption);
+            command.AddOption(PackageDirOption);
+            command.AddOption(InteractiveOption);
+            command.AddOption(PrereleaseOption);
+            command.AddOption(CommonOptions.DiagOption());
```

Comment:
The current `dotnet add package` command doesn't have `--diagnostics` option. No references to this option in NuGet side [implementation](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/AddPackageReferenceCommand.cs) for this command. I see that this is one of the common options. Is this something new that we should support?

Reply:
Nope, we don't want to change anything that will require changes on your side in this PR, so I'll remove it. Thanks for pointing this out.